cypress/e2e/builder/enterSettings.cy.ts

```diff
@@ -38,6 +38,12 @@ describe('Enter Settings', () => {
       .type('Title');
 
     cy.get(buildDataCy(SAVE_TITLE_BUTTON_CY)).click();
+
+    cy.get(buildDataCy(SAVE_TEXT_BUTTON_CY)).should('be.disabled');
```

Comment:
Are you sure this is the button you want to test ? Isn't it `SAVE_TITLE_BUTTON_CY` instead of `SAVE_TEXT_BUTTON_CY` ?

Reply:
Ah yes you're right !